expr_insecure_test.go

```diff
@@ -192,20 +192,6 @@ func TestExprInsecureDetectUntrustedValue(t *testing.T) {
 				"github.event.pages.*.page_name",
 			},
 		},
-		testCase{
-			"contains(github.event.pages.*.page_name, github.event.issue.title)",
-			[]string{
-				"github.event.pages.*.page_name",
-				"github.event.issue.title",
-			},
-		},
-		testCase{
-			"contains(github.event.*.body, github.event.*.*)",
-			[]string{
-				"github.event.",
-				"github.event.",
-			},
-		},
```

Comment:
Instead of removing these tests, would you use some other unsafe functions instead of `contains`? These cases test insecure properties usage inside function calls.

Reply:
Sure, replaced `contains` with `fromJson`